cmd/soroban-rpc/internal/jsonrpc.go

```diff
@@ -206,6 +206,13 @@ func NewJSONRPCHandler(cfg *config.Config, params HandlerParams) Handler {
 			queueLimit:           cfg.RequestBacklogGetTransactionQueueLimit,
 			requestDurationLimit: cfg.MaxGetTransactionExecutionDuration,
 		},
+		{
+			methodName:           "getTransactions",
+			underlyingHandler:    methods.NewGetTransactionsHandler(params.Logger, params.LedgerReader, params.LedgerEntryReader, cfg.MaxTransactionsLimit, cfg.DefaultTransactionsLimit, cfg.NetworkPassphrase),
+			longName:             "get_transactions",
+			queueLimit:           cfg.RequestBacklogSendTransactionQueueLimit,
```

Comment:
just driving by, noticed `getTransactions` is applying existing tx sub queue and duration limit cfg values, should it be using a new pair of queue/duration limit config keys, e.g. `cfg.RequestBacklogGetTransactionsQueueLimit` , it seems like the request/load profiles would be different.

Reply:
Ah good catch. Yes it should be using a different config separate to this endpoint